packages/roosterjs-editor-types/lib/Constants.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Do not modify any code from old packages (roosterjs-editor-*), we will delete those old code files soon.

If need, add them into new packages.

And do not add consts to type package. Type package is suppose to generate 0 js code. It is just type declaration.